vm/src/builtins/pytype.rs

```diff
@@ -517,6 +517,47 @@ impl PyType {
             "Setting __dict__ attribute on a type isn't yet implemented".to_owned(),
         ))
     }
+
+    #[pyproperty(magic)]
+    fn text_signature(&self) -> Option<String> {
+        let doc_string = self.get_attr("__doc__");
```

Comment:
If you are interested in, I feel like `tp_doc` which should be added in `PyTypeSlots` is related to improvement of this line.

Reply:
I tried to apply your suggestion in 78ff936. Could you check that it was you said?